doc/user/release-notes.html

```diff
@@ -54,6 +54,7 @@ <h2>Release notes for Crux</h2>
 
 <h4>Minor changes</h4>
 <ul>
+  <li>7 July 2023: Fixed several bugs related to spectral counts. Spectral counts can now parse native Percolator output. Fixed issue #639 and #630.</li>
```

Comment:
I don't think we should refer to issue numbers in the release notes.  Can we just delete that sentence? Or add a brief description of the bugs.

Reply:
Sure I can delete it.